app/views/candidate_interface/sign_up/new.html.erb

```diff
@@ -11,26 +11,14 @@
         <%= t('authentication.sign_up.heading') %>
       </h1>
 
-      <%= f.govuk_email_field :email_address, label: { text: t('authentication.sign_up.email_address.label'), size: 'm' }, hint: { text: t('authentication.sign_up.email_address.hint_label') }, width: 'two-thirds', autocomplete: 'email', spellcheck: false %>
-
-      <h2 class="govuk-heading-m">You will not need a password to use this service</h2>
-      <p class="govuk-body">Instead, you’ll sign in using your email address. Each time you sign in, we will send you a link so you can return to your application.</p>
-
-      <h2 class="govuk-heading-m">How we look after your data</h2>
-      <p class="govuk-body">Your data will be looked after by the Department for Education and the training providers you apply to.</p>
-      <p class="govuk-body">We use it to process your application, build a better service and improve teacher recruitment and retention.</p>
-      <p class="govuk-body govuk-!-margin-bottom-6">Take a look at our <%= govuk_link_to 'privacy policy', candidate_interface_privacy_policy_path %> before starting your application to check you understand how we use your data.</p>
-
-      <%= f.govuk_check_boxes_fieldset :accept_ts_and_cs, legend: { text: 'Terms of use' } do %>
-        <p class="govuk-body">Our <%= govuk_link_to 'terms of use', candidate_interface_terms_path %> contain information about:</p>
-        <ul class="govuk-list govuk-list--bullet">
-          <li>the application process</li>
-          <li>contacting us</li>
-          <li>checking you’re safe to work with children</li>
-        </ul>
-
-        <%= f.govuk_check_box :accept_ts_and_cs, 'true', multiple: false, label: { text: t('authentication.sign_up.accept_terms_checkbox') }, link_errors: true %>
-      <% end %>
+      <%= f.govuk_email_field :email_address, label: { text: t('authentication.sign_up.email_address.label'), size: 's' }, hint: { text: t('authentication.sign_up.email_address.hint_label') }, width: 'two-thirds', autocomplete: 'email', spellcheck: false %>
+
+      <p class="govuk-body">Your email address will be looked after by the Department for Education and shared with the training providers you apply to.</p>
```

Comment:
This bit doesn't make sense to me. If we're saying a thing about privacy we shouldn't limit it to email address.

Reply:
I thought it might be more contextually relevant than "data" which always feels a bit jargon-y to me? Perhaps we could say "Your email address and other personal information you enter" or similar?

Ideas welcome!

Reply:
Your email address and other personal information you enter as part of your application.